rust/operator-binary/src/listener_controller.rs

```diff
@@ -269,6 +272,7 @@ pub async fn reconcile(listener: Arc<Listener>, ctx: Arc<Ctx>) -> Result<control
         })?;
 
     let nodes: Vec<Node>;
+    let kubernetes_service_fqdn: String;
```

Comment:
Getting C vibes ^^

Reply:
Yuup, they've gotta be stored somewhere though.